spec/spec_helper.rb

```diff
@@ -18,6 +18,7 @@
   c.hook_into :webmock
   c.debug_logger = File.open('log/vcr.log', 'w')
   c.configure_rspec_metadata!
+  c.allow_http_connections_when_no_cassette = true
```

Comment:
This isn't the way to deal with VCR errors. What's happening is VCR is telling you "hey, a spec is trying to access the network, which isn't ok. Please fix it." and this code change is answering "that's ok, just go ahead and access the network".

On top of that, the network activity which is happening is kinda interesting. The code is making requests to 169.254.169.254, which is a "dynamically configured link-local addresses", which is private to the local network and won't ever be accessible over the internet like other IP addresses are. This [server fault Q&A](http://serverfault.com/questions/427018/what-is-this-ip-address-169-254-169-254) explains the situation quite well. This specific IP address is used internally by AWS. All-in-all, I think something is not right here; maybe the use of the S3 gem's interface. I'd happily debug this, but I need the access creds as requested above.

To be clear though: we need to remove this from the VCR config. I don't want our specs taking to the network during their run.